registry_no_code_module.go

```diff
@@ -68,6 +67,9 @@ type RegistryNoCodeModuleCreateWorkspaceOptions struct {
 
 	// SourceUrl is the URL of the source of the workspace.
 	SourceURL *string `jsonapi:"attr,source-url,omitempty"`
+
+	// ExecutionMode is the execution mode of the workspace.
+	ExecutionMode *string `jsonapi:"attr,execution-mode,omitempty"`
```

Comment:
I think you'll also need to include `AgentPoolID` too, in case someone uses `agent` for its `ExecutionMode`?